pkg/webui/components/footer/index.js

```diff
@@ -51,12 +57,18 @@ const Footer = ({ className, documentationLink, links, supportLink, onlineStatus
       <span className={style.version}>v{process.env.VERSION}</span>
       {documentationLink && (
         <Link.Anchor className={style.documentation} secondary href={documentationLink} external>
-          <Message content={sharedMessages.documentation} />
+          <Message
+            content={
+              isMinimized && supportLink
+                ? sharedMessages.documentationAbbreviation
+                : sharedMessages.documentation
```

Comment:
Let's not use an abbreviation here. It might work in English but might be troublesome in other languages. Let's instead hide the version tag. Use a tight media query for this so it only get's hidden when it becomes necessary.